fauna/client/client.py

```diff
@@ -375,6 +389,27 @@ def _query(
           schema_version=schema_version,
       )
 
+  def stream(
```

Comment:
We don't have to do it in this PR, but before we release, please document this method